lib/spoom/cli/srb/sigs.rb

```diff
@@ -53,6 +53,118 @@ def strip(*paths)
           say("Stripped signatures from `#{transformed_files}` file#{transformed_files == 1 ? "" : "s"}.")
         end
 
+        # Extract signatures from gem files and save them to the output file
```

Comment:
```suggestion
        # Extract signatures from gem's files and save them to the output file
```
I think it'll help with clarity since we don't want them to think `Gemfile`.